mapit/models.py

```diff
@@ -386,7 +386,7 @@ class CodeType(models.Model):
     # This could be extended to a more generic data store of information on an
     # object, perhaps.
 
-    code = models.CharField(max_length=10, unique=True, help_text="A unique code, eg 'ons' or 'unit_id'")
+    code = models.CharField(max_length=100, unique=True, help_text="A unique code, eg 'ons' or 'unit_id'")
```

Comment:
As type.code and code.code both went up to 500 when they were migrated, shall we just do the same here for consistency? And we might as well do nametype.code up to 500 as well, then they all are :)